test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,185 @@
+
+
+metrics_dict = {
+    'Recall': recall_score,
+    'Recall Bounds': recall_wilson,
+    'Recall Error': recall_normal_err,
+    'Accuracy': accuracy_score,
+    'Accuracy Bounds': accuracy_wilson,
+    'Accuracy Error': accuracy_normal_err,
+    'Recall Bounds Flipped': incorrect_flipped_recall_wilson,
+    'Recall Bounds Single': incorrect_single_bound_recall_wilson,
+    'Recall Error Negative': incorrect_negative_recall_normal_err,
+    'Recall Error Single': incorrect_single_error_recall_normal_err,
+}
+
+
+@pytest.fixture()
+def metric_frame():
+    return MetricFrame(
+        metrics=metrics_dict,
+        y_true=y_t,
+        y_pred=y_p,
+        sensitive_features=g_1)
+
+
+def test_plotting_output(metric_frame):
+    axs = plot_metric_frame(metric_frame)
+    assert len(axs) == 4
+    assert isinstance(axs[0], matplotlib.axes.Axes)
+
+
+def test_ambiguous_error_metric(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=["Recall"], error_bars=[
+                          "Recall Error"], conf_intervals=["Recall Bounds"])
+        assert str(exc.value) == _GIVEN_BOTH_ERROR_BARS_AND_CONF_INT_ERROR
+
+
+def test_unequal_error_bars_length(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=[
+                          "Recall", "Accuracy"], error_bars=["Recall Error"])
+        assert str(exc.value) == _METRIC_AND_ERROR_BARS_NOT_SAME_LENGTH_ERROR
+
+
+def test_unequal_conf_intervals_length(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=[
+                          "Recall", "Accuracy"], conf_intervals=["Recall Bounds"])
+        assert str(exc.value) == _METRIC_AND_CONF_INTERVALS_NOT_SAME_LENGTH_ERROR
+
+
+def test_flipped_bounds(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=["Recall"],
+                          conf_intervals=["Recall Bounds Flipped"])
+        assert str(exc.value) == _CONF_INTERVALS_FLIPPED_BOUNDS_ERROR
+
+
+def test_single_bound(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=["Recall"],
+                          conf_intervals=["Recall Bounds Single"])
+        assert str(exc.value) == _CONF_INTERVALS_MUST_BE_TUPLE
+
+
+def test_negative_error(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=["Recall"],
+                          error_bars=["Recall Error Negative"])
+        assert str(exc.value) == _ERROR_BARS_NEGATIVE_VALUE_ERROR
+
+
+def test_single_error_bar(metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(metric_frame, metrics=["Recall"],
+                          conf_intervals=["Recall Error Single"])
+        assert str(exc.value) == _ERROR_BARS_MUST_BE_TUPLE
+
+
+def test_single_ax_input(metric_frame):
```

Comment:
Can we add one for multiple, too?

Reply:
doesn't make sense anymore since we don't take ax input